src/client/cli.ts

```diff
@@ -0,0 +1,41 @@
+import yargs from 'yargs'
+import { hideBin } from 'yargs/helpers'
+
+import { SocialInboxClient } from './index.js'
+import { generateKeypair } from 'http-signed-fetch'
+import { DEFAULT_PUBLIC_KEY_FIELD } from '../server/apsystem.js'
+
+// https://github.com/yargs/yargs/issues/225
+const argv = yargs(hideBin(process.argv))
+  .usage('usage: $0 <account>')
```

Comment:
the reason I didn't use the .command API yet is because I don't think it's possible to set up as `cli <email> <command>` and i figured that because this is an internal tool, leaving it without that implemented yet until it's needed that it does more stuff is ok. check out the issue i linked.

but.. i'm not strongly opinionated about the CLI. if you can suggest a CLI that better adapts to what yargs provides i'll happily implement it

Reply:
I think it'd be fine if the email was passed in as an environment variable or a command line flag, personally. I found the `.command` syntax to be convenient to use for other projects.